R/get_generic.R

```diff
@@ -0,0 +1,72 @@
+## function name get_generic
+
+#' retrieve all parameters of specified type
+#' @param data parameter dataframe outputted from load_epidata()
+#' @param parameter_name name of the parameter type to retrieve, ensuring the name matches that in data
+#' @return dataframe with all parameter estimates and specified columns
+#' @examples
+#' df <- load_epidata(pathogen = "ebola")
+#' get_generic(data = df$params, parameter_name = "Human delay - serial interval")
+#' @examples
+#' df <- load_epidata(pathogen = "marburg")
+#' get_generic(data = df$params, parameter_name = "Attack rate")
+get_generic <- function(data, parameter_name) {
```

Comment:
We need to implement some checks on the inputs. Can you take a look at the code in https://github.com/mrc-ide/epireview/commit/0beee80bb8bff860e5016c4c7381b72be9784622 and see if you could implement that here?

Reply:
Will ask someone to help with this